kong/plugins/request-termination/schema.lua

```diff
@@ -32,6 +34,11 @@ return {
           and not is_present(config.body) then
             return nil, "content_type requires a body"
           end
+          if config.echo and (
```

Comment:
I think we can replace this logic with the `mutually_exclusive_sets` entity validator. For example:
`{ mutually_exclusive_sets = { { "echo" }, { "content_type", "body" } }`

Reply:
That would only make sense if we'd replace the entire custom validator with standard entity_checks, I spend 2 hours on it, but it so overwhelmingly under-documented that I couldn't get it to work. So I'm giving up on that. Someone else may send in a separate PR to fix it. Not in scope of this change.

Reply:
Agreed on not touching unrelated parts of the validator. However, if the new field can leverage the validator, I think we should do it (leaving it to later creates another breaking change in the future).

Reply:
Actually, @Tieske, the `mutually_exclusive_sets` entity check can't be used here (the `echo` field is required and has a default value).